app/services/charts_layout_service.rb

```diff
@@ -29,10 +29,21 @@ def build_charts
     YAML.load(File.open(find_chart_path))
   end
 
+  def override_class_name
+    class_path = @perf_record.class.name.split('::')
+
+    # Overide all ManageIQ Providers ContainerManager to point to the default
+    # provider chart yaml file, ManageIQ_Providers_Default_ContainerManager.yaml
+    class_path[2] = "Default" if
+      @perf_record.kind_of?(ManageIQ::Providers::ContainerManager)
```

Comment:
Why is ContainerManager different?  Can we ask an object for the path to the object's chart yaml? Can this ChartLayoutService have a decorator for the real class (or something similar) that can answer this question?

I don't really know but it feels like this is a common problem for locating various paths for different types of objects in the UI.